p2p/p2p.go

```diff
@@ -103,6 +107,45 @@ func NewTCPNode(ctx context.Context, cfg Config, key *ecdsa.PrivateKey, connGate
 	return tcpNode, nil
 }
 
+// externalMultiAddrs returns the external IP and Hostname fields as multiaddrs using the listen TCP address ports.
+func externalMultiAddrs(cfg Config) ([]ma.Multiaddr, error) {
+	tcpAddrs, err := cfg.ParseTCPAddrs()
+	if err != nil {
+		return nil, err
+	}
+
+	var ports []int
+	for _, addr := range tcpAddrs {
+		ports = append(ports, addr.Port)
+	}
+
+	var resp []ma.Multiaddr
+
```

Comment:
nit: can remove newline

Reply:
added new line since we append to it in two different blocks